CSS/styles.css

```diff
@@ -77,17 +113,64 @@ ol{
 }
 
 .rules-container{
-    border : 3px solid yellow;
+    /* border : 3px solid yellow; */
     display : flex;
     flex-direction: column;
     gap : 3rem;
     width : 60%;
 }
 
+.categories-flex-container{
+    /* border : 3px solid yellow; */
+    display : flex;
+    flex-direction: row;
+    justify-content : center;
+    gap : 2rem;
+}
 
+.single-category-container{
+    /* border : 3px solid green; */
+    display : flex;
+    flex-direction: column;
+    padding : 1.2rem;
+    width : 35rem;
+    gap : 1rem;
+    border-radius: 0.8rem;
+    background-color : var(--light-test-color);
+    cursor : pointer;
+    /* height : 25rem; */
+}
 
+.img-container{
+    /* border : 3px solid purple; */
+    width : 100%;
+    height : 20rem;
+}
 
+.img-container img{
+    border-radius: 0.8rem;
+}
 
+.category-desc{
+    /* border : 3px solid purple; */
+    border-radius: 0.8rem;
+    padding : 0.8rem;
+    background-color : var(--background-color);
+}
 
-
-
+.question-container{
+    /* border : 3px solid blue; */
+    width : 60%;
+    max-width: 60rem;
+    padding : 1rem;
+    border-radius: 1rem;
+    background-color : var(--light-test-color);
```

Comment:
here you used "--light-test-color" it should be named any understandable name. I am not this is your final color naming or not. Other than that everything is clean and simple. Great work